src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -412,10 +412,10 @@ public function column_source( $post ) {
 			$html[] = $this->get_status_icon( $post );
 		}
 
-		if ( 'ea/ics' === $post->post_mime_type || 'ea/csv' === $post->post_mime_type ) {
+		if ( in_array( $record->origin, array( 'ics', 'csv' ) ) ) {
 			$html[] = '<p><b>' . esc_html( $record->meta['source_name'] ) . '</b></p>';
 		} else {
-			$html[] = '<p><b><a href="' . esc_url( $record->meta['source'] ) . '">' . esc_html( $record->meta['source_name'] ) . '</a></b></p>';
+			$html[] = '<p><b><a href="' . esc_url( $record->meta['source'] ) . '" target="_blank">' . esc_html( $record->meta['source_name'] ) . '</a></b></p>';
```

Comment:
Since this is `target="_blank"`, let's add the following inside the anchor tag.

```
<span class="screen-reader-text">' . __( ' (opens in a new window)' ) . '</span>
```